bigtable/client/filters.h

```diff
@@ -0,0 +1,309 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_FILTERS_H_
+#define GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_FILTERS_H_
+
+#include <bigtable/client/version.h>
+#include <google/bigtable/v2/data.pb.h>
```

Comment:
Are there multiple groups of headers (e.g., Bigtable, generated, third-party, STL)? Should they be each separated from the next group with blank lines? In some files, that is the case, in others (such as this one), it's not. This needs to be standardized.

Reply:
Fixed.  And yes it should be part of the guidelines.